src/main/webapp/availabilityTable.jsp

```diff
@@ -37,13 +37,13 @@
         </c:otherwise>
       </c:choose>
             <td>${pageScope.list.get(0).get(i).time()}</td>
-              <c:forEach items = "${pageScope.list}" var = "day">
-                <td onmouseenter="event.preventDefault(); toggleTile(this);"
-                    onmousedown="event.preventDefault(); markMouseDown(); toggleTile(this);"
-                    data-utc="${day.get(i).utcEncoding()}" 
-                    class="${day.get(i).getClassList()}">
-                </td>
-              </c:forEach>
+            <c:forEach items = "${pageScope.list}" var = "day">
+              <td onmouseenter="event.preventDefault(); toggleTile(this);"
+                  onmousedown="event.preventDefault(); markMouseDown(); toggleTile(this);"
+                  data-utc="${day.get(i).utcEncoding()}" 
+                  class="${day.get(i).getClassList()}">
+              </td>
+            </c:forEach>
```

Comment:
l thought these would go on the same level as `</c:choose>` (line 38). Not sure though.

Reply:
It looks like the indentation doesn't effect the performance at all, so I'll see what Alex thinks makes the most sense. I'm just weirded out by the tr tags not lining up.